openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>[enhancement]</p><p> </p><p>What would be super useful here is some kind of rough table of how the protocol works like</p><p> </p><ol><li>Input handling using gufe</li><li>Parameterization using OpenMMForceFields &amp; OpenFF</li><li>OpenMM object creation</li><li>Minimization</li><li>NVT equilibration (if not gas phase)</li><li>NPT equilibration (if not gas phase)</li><li>NPT production (if not gas phase)</li></ol><p> </p><p>The table could have the columns "step", "software used", and "outputs". That way folks know what files come out of which steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Optional: can we add a simple complex picture here?</p><p>I could make one for you :)</p>

Reply:
<p>Added a table describing these steps.</p>